modules/core/src/test/scala/com.snowplowanalytics.snowplow.lakes/TestSparkEnvironment.scala

```diff
@@ -28,9 +28,15 @@ case class TestSparkEnvironment(
 
 object TestSparkEnvironment {
 
-  def build(windows: List[List[TokenedEvents]]): Resource[IO, TestSparkEnvironment] = for {
+  sealed trait Target
+  case object Delta extends Target
+  case object Hudi extends Target
+  case object Iceberg extends Target
+
+  def build(target: Target, windows: List[List[TokenedEvents]]): Resource[IO, TestSparkEnvironment] = for {
     tmpDir <- Resource.eval(IO.blocking(Files.createTempDirectory("lake-loader")))
```

Comment:
Not related to this change, but maybe we could use here: `fs2.io.file.Files.tempDirectory`? Because at the moment this temp directory is not cleaned when test finishes, right?